Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/Handlers/INavigationHandler.cs

```diff
@@ -1,10 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.InputSystem.EventData;
+using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Input;
 using UnityEngine.EventSystems;
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem.InputHandlers
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers
 {
     /// <summary>
     /// Interface to implement to react to navigation gestures.
```

Comment:
MRTK Interface names should be prefixed with either IMixedReality or IMixedRealityToolkit

Reply:
done